README.md

```diff
@@ -32,12 +32,14 @@ The firmware will be flashed on to your connected device. You can watch the prog
 
 ESP Launchpad also lets you easily publish your firmware apps for others to try.
 
-The ESP Launchpad Quick Start page would be rendered by referring to a TOML configuration file, where you can configure where to pick all the component images of your firmware, and the supported hardware. You could also link to any support phone apps to work along with this firmware. A sample TOML config file can be viewed here
+The ESP Launchpad Quick Start page would be rendered by referring to a TOML configuration file, where you can configure where to pick all the component images of your firmware, and the supported hardware. You could also link to any support phone apps to work along with this firmware and include additional information about your application in **Markdown format** within the TOML config file itself. ESP Launchpad will then render the application information and any additional instructions in the browser after flashing the firmware image. A sample TOML config file can be viewed [here](https://github.com/espressif/esp-launchpad/blob/main/config/config.toml)
```

Comment:
typo : link to any supported phone apps instead of support phone apps.
 - also break down the sentence. Start the next on a new line - You can also include additional information ....